example4.py

```diff
@@ -0,0 +1,15 @@
+'''Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов. 
+Сколько журавликов сделал каждый ребенок, если известно, что Петя и Сережа сделали одинаковое количество журавликов, 
+а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+
+*Пример:*
+
+6 -> 1  4  1
+24 -> 4  16  4
+    60 -> 10  40  10'''
+
+a = int(input())
+k = int((a/3)*2)
+p = int((k/2)/2)
+s = int(p)
```

Comment:
Приоритет у операций умножения и деления одинаковый.
```suggestion
k = a // 3 * 2
p = s = k // 2 // 2  # Или сразу делить на 4
```